docs/deployment/docker-deployment.md

```
@@ -0,0 +1,227 @@
+
+Pull the latest WandererNotifier image:
+
+```bash
+docker pull wanderernotifier/app:latest
+```
+
+### Step 2: Create an Environment File
+
+Create a `.env` file with the following structure:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)